### DISCUSSION TITLE
using the 'archive' page as my homepage

### DISCUSSION THREAD

**Discussion by ghost** [jpanther/congo#537] [Q&A] [answered]
Hi,

I currently use the recent articles partial to generate a list of a small posts. (https://mark.hysted.com)

I would actually like to show on the homepage the list in the format shown in the archive page (https://mark.hysted.com/posts/) as I like to show the pagination at the bottom of the lists of posts - more 'normal' an approach to me.  Seems a bit strange to have a list of posts with no obvious way to go back in the list - seems to be a Hugo thing.

Anyone tried this in the Congo theme?

**Comment by ghost** [comment, marked as answer]
tried adding {{ partial "pagination.html" . }} into page.html and that worked.  I am sorted.

**Reply by SathyaBhat** [reply]
Thanks @hystedm for this tip. I wanted the same for my home page, I switched over to custom adding the pagination partial and it did the job!
